dspace/config/spring/api/ads-integration.xml

```diff
@@ -0,0 +1,122 @@
+        <entry key-ref="ads.doi" value-ref="adsDoiContrib"/>
+        <entry key-ref="ads.type" value-ref="adsTypeContrib"/>
+        <entry key-ref="ads.id" value-ref="adsIdContrib"/>
+        <entry key-ref="ads.issue" value-ref="adsIssueContrib"/>
+        <entry key-ref="ads.keywrd" value-ref="adsKeywordContrib"/>
+        <entry key-ref="ads.lang" value-ref="adsLangContrib"/>
+        <entry key-ref="ads.source" value-ref="adsSourceContrib"/>
+        <entry key-ref="ads.title" value-ref="adsTitleContrib"/>
+    </util:map>
+
+    <bean id="adsTitleContrib" class="org.dspace.importer.external.metadatamapping.contributor.SimpleJsonPathMetadataContributor">
+        <property name="field" ref="ads.title"/>
+        <property name="query" value="$.title"/>
+    </bean>
+    <bean id="ads.title" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.title"/>
+    </bean>
+
+    <bean id="adsSourceContrib" class="org.dspace.importer.external.metadatamapping.contributor.SimpleJsonPathMetadataContributor">
+        <property name="field" ref="ads.source"/>
+        <property name="query" value="$.pub"/>
+    </bean>
+    <bean id="ads.source" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.source"/>
+    </bean>
+
+    <bean id="adsLangContrib" class="org.dspace.importer.external.metadatamapping.contributor.SimpleJsonPathMetadataContributor">
+        <property name="field" ref="ads.lang"/>
+        <property name="query" value="$.lang"/>
+    </bean>
+    <bean id="ads.lang" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.language.iso"/>
+    </bean>
+
+    <bean id="adsKeywordContrib" class="org.dspace.importer.external.metadatamapping.contributor.SimpleJsonPathMetadataContributor">
+        <property name="field" ref="ads.keywrd"/>
+        <property name="query" value="$.keyword"/>
+    </bean>
+    <bean id="ads.keywrd" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.subject"/>
+    </bean>
+
+    <bean id="adsIssueContrib" class="org.dspace.importer.external.metadatamapping.contributor.SimpleJsonPathMetadataContributor">
+        <property name="field" ref="ads.issue"/>
+        <property name="query" value="$.year"/>
+    </bean>
+    <bean id="ads.issue" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.date.issued"/>
+    </bean>
+
+    <bean id="adsIdContrib" class="org.dspace.importer.external.metadatamapping.contributor.SimpleJsonPathMetadataContributor">
+        <property name="field" ref="ads.id"/>
+        <property name="query" value="$.bibcode"/>
+    </bean>
+    <bean id="ads.id" class="org.dspace.importer.external.metadatamapping.MetadataFieldConfig">
+        <constructor-arg value="dc.identifier.adsbibcode"/>
```

Comment:
This field doesn't exist in DSpace.  Should it just be set to `dc.identifier.other`? or something similar?